src/Symfony/Component/Console/Application.php

```diff
@@ -588,7 +588,10 @@ public function renderException(\Exception $e, OutputInterface $output)
         $output->writeln('', OutputInterface::VERBOSITY_QUIET);
 
         do {
-            $title = sprintf('  [%s]  ', get_class($e));
+            $title = $output->isVerbose() && ($code = $e->getCode()) > 0
+                ? sprintf('  [%s - code %s]  ', get_class($e), $code)
+                : sprintf('  [%s]  ', get_class($e))
+            ;
```

Comment:
I propose to change 2 things here:

 * Use `0 !== $code` instead of `$code > 0`, just anything that's not the default value should be printed imo
 * Use just one sprintf call:

   ```php
   $title = sprintf('  [%s%s]', get_class($e), $output->isVerbose() && ($code = $e->getCode()) ? ' - code '.$code : '')
   ```